gen3-integration-tests/tests/test_gen3_workflows.py

```diff
@@ -495,7 +495,7 @@ def test_nextflow_workflow(self):
 
             # Verify the expected file is non-empty
             response = self.gen3_workflow.get_bucket_object_with_boto3(
-                object_path=f"{self.s3_storage_config.bucket_name}/{expected['filename']}",
+                object_path=f"{task['workDir']}/{expected['filename']}",
```

Comment:
you changed this in one place but kept `self.s3_storage_config.bucket_name` everywhere else - is there a reason it has to be different here?